### ISSUE TITLE
indent for class bodies

### ISSUE THREAD

**Issue by a-x-** [babel/eslint-plugin-babel#175]
Can indent rule for experimental features be added here?

eslint core will not support it.
look for details: https://github.com/eslint/eslint/issues/11719

copy of original issue:

**environment**

* **ESLint Version:** v5.16.0
* **Node Version:** v11.12.0
* **npm Version:** 6.9.0

**parser**: babel-eslint

**minimal configuration:**

<details>
<summary>Configuration</summary>

```js
module.exports = {
	parser: 'babel-eslint',
	parserOptions: {
		ecmaVersion: 6,
		sourceType: 'module',
		ecmaFeatures: { },
	},

	settings: { },
	rules: {
		'indent': ['warn', 2, { SwitchCase: 1 }]
	},
};
```

</details>

**source code**

```js
// works in functions:
// function Foo () {
// 				cellContentProps = { onStatusChange: this.props.onStatusChange }
// }
// --> warning  Expected indentation of 2 spaces but found 4 tabs

// does not works in class bodies:
export default class Foo {
          cellContentProps = { onStatusChange: this.props.onStatusChange }
}
// --> no warnings
```

```bash
eslint path/to/file.js
```

**expected**
`warning  Expected indentation of 2 spaces but found 4 tabs`

**pull request?** maybe I can create PR, if you describe all the details how and where to fix

**Comment by shmup** [comment]
Hey! We still need this. It's 2020 and this will still pass indentation rules expecting `2`:

```
class Foo {
    scream() {
      console.debug('ahhhhh');
    }
}
```

**Comment by existentialism** [comment]
@shump the original issue relates to the `indent` rule not working with class fields?

Trying your code locally, the `indent` rule seems to trigger just fine? Can you create a small repo that reproduces your problem?

**Comment by kaicataldo** [comment]
Thank you for the issue. Now that [@babel/eslint-plugin has been released](https://babeljs.io/blog/2020/07/13/the-state-of-babel-eslint), we are making this repository read-only. If this is a change you would still like to advocate for, please reopen this in the [babel/babel monorepo](https://github.com/babel/babel).
